### PR TITLE
Update PPP and project status

### PR DESCRIPTION
**Pull request by JordanChua**
- Update my own PPP #154 
- Fix minor bug in project status error message by changing `OR` to `or`

### CODE REVIEW

**Review by SweetPotato0213 — DISMISSED** [review]
LGTM!

**Review by SweetPotato0213 — APPROVED** [review]
LGTM!

File: docs/team/jordanchua.md [hunk 1]
```diff
@@ -9,19 +9,79 @@ JerylFypManager is a desktop application catered to professors to manage and tra
...
+      of any FYP project so that he/she can identify which projects may need more attention or help, especially
+      those that are `YTS` or `IP`.
+  * Highlights:
+    * This enhancement was a totally different command by itself, hence it was challenging to implement everything
+      from scratch. It required an in-depth analysis of the current code base, and I had to do extensive
+      code-tracing to implement the feature.
+    * It was also hard to implement the status as a small coloured box at the top right hand corner
+      of the Project card displayed in the UI since I was not too familiar with FXML.
+  * Credits:
+    * Would like to thank @russeldash332 for the help in implementing the status as a coloured box
```
Comment (RussellDash332) [inline docs/team/jordanchua.md:36]: Whoops you're missing an L here haha

File: src/main/java/jeryl/fyp/model/student/ProjectStatus.java [hunk 2]
```diff
@@ -9,7 +9,7 @@
  */
 public class ProjectStatus {
     public static final String MESSAGE_CONSTRAINTS =
-            "Status should only be of the form YTS, IP OR DONE";
+            "Status should only be of the form YTS, IP or DONE";
```
Comment (RussellDash332) [inline src/main/java/jeryl/fyp/model/student/ProjectStatus.java:12]: I thought this cannot be changed? Not a bug so

Comment (JordanChua) [reply src/main/java/jeryl/fyp/model/student/ProjectStatus.java:12]: oof we can't change this? :((1

File: docs/team/jordanchua.md (hunk 1 above)
Comment (JordanChua) [reply docs/team/jordanchua.md:36]: OH SHIT PS

**Pull request merged**